api/src/main/java/ca/bc/gov/educ/api/dataconversion/process/StudentProcess.java

```diff
@@ -282,9 +283,7 @@ private Pair<ConversionResultType, Boolean> handleProgramCode(String programCode
 
     private ConversionResultType processSccpFrenchCertificates(GraduationStudentRecord student, ConversionStudentSummaryDTO summary) {
         if (StringUtils.equals(student.getProgram(), "SCCP")
-            && ( StringUtils.isNotBlank(student.getSchoolOfRecord())
-                 && student.getSchoolOfRecord().startsWith("093") )
-        ) {
+            && isSchoolForProgramFrancophone(student.getSchoolOfRecordId())) {
```

Comment:
Do we need the null check still?

Reply:
It is not supposed to have null schoolOfRecordId in GraduationStudentRecord.   If so, that is critical, and intentionally good to have NPE instead of moving forward with the wrong grad program.  Anyway, I will add the null check here, and monitor in PROD that schoolOfRecordId is null to have the wrong grad program in student load & ongoing updates.